### PR TITLE
{math}[iimpi/2019a] Ray v2.3.1

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2125.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/68a40801d7039b094a3cbf1dab9bccf3 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2545.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/e9b90b4833425b11859eaf7823d38f3e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3171.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/5d6a32cd7b91c98d653f21a9cfc54185 for a full test report.

File: easybuild/easyconfigs/r/Ray/Ray-2.3.1-iimpi-2019a.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    'files': ['bin/Ray'],
+    'dirs': [],
+}
+
+sanity_check_commands = [
+    "mpirun -n 1  %(installdir)s/bin/Ray -o test -p test_1.fastq test_2.fastq -k 31",
+    "mpirun -n 1 Ray -help",
+]
+
+moduleclass = 'math'
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/r/Ray/Ray-2.3.1-iimpi-2019a.eb]: `bio`?

Comment (boegel) [reply easybuild/easyconfigs/r/Ray/Ray-2.3.1-iimpi-2019a.eb]: fixed in 2b214df

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
sms - Linux centos linux 7.6.1810, AMD EPYC 7601 32-Core Processor, Python 2.7.5
See https://gist.github.com/50ce4b37927c3c65df1a606dde91d1ae for a full test report.

**Comment by migueldiascosta** [comment]
Going in, thanks @boegel!

**Pull request merged**